src/patternfly/components/NotificationBadge/examples/NotificationBadge.md

```diff
@@ -52,6 +52,50 @@ import './NotificationBadge.css'
 </div>
 ```
 
+### Expanded
+
+```hbs
+<div class="pf-t-dark">
+{{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Notifications" aria-expanded="true"'}}
```

Comment:
Were you using `aria-expanded="true"` here instead of `button--IsExpanded="trye"` to avoid applying `pf-m-expanded` to the button itself?

Reply:
Yeah, it's only supported on the control button, and noticed the notification badge button in the masthead was using the expanded class, so updated that just to be `aria-expanded`, too. Looking at the react component, I think we'll want to add that there, too - https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/NotificationBadge/NotificationBadge.tsx